.eslintrc

```diff
@@ -30,7 +30,6 @@
         "argsIgnorePattern": "^_"
       }
     ],
-    "import/extensions": ["error", "always", {"ignorePackages": true} ],
```

Comment:
@bhouston this is kind of a difficult rule to follow, it means you need to add the file extension whenever you import in typescript.

Is it ok to get rid of this rule?  Makes the code easier to write/read when you can just do:

```ts
import { useRegistry } from './hooks/useRegistry';
```

wheras before you would need to do:

```ts
import { useRegistry } from './hooks/useRegistry.js';
```

Reply:
I did this because it allows for many loaders to load JavaScript files in the browser without bundling.  This explains the reasoning: https://stackoverflow.com/questions/70461497/why-do-i-need-to-add-the-js-extension-when-importing-files-in-typescript-but-no?utm_source=feedburner&utm_medium=email

That said, I can just always use a bundler.

On that topic, right now if there are errors in the code, say when I run exec-graph, I do not see any source maps.  It makes it a bit harder to track down the errors to the original line in the typescript file.

Reply:
When would you want to load js files in the browser?
Yes seeing source maps would be ideal.  A nice way to do this is to just use `ts-node` - then you get all errors showing up in the original source code location.  And you get rid of the need for the step to build the files before.  I originally wanted to setup running the exec-graph scripts using ts-node, but it will not work when you want to import `.js` files:
https://github.com/TypeStrong/ts-node/issues/783